spec/features/consumer/multilingual_spec.rb

```diff
@@ -84,21 +83,29 @@
       end
     end
 
-    it "allows switching language via the main navigation" do
-      visit root_path
+    context "when there are multiple languages available" do
+      before do
+        allow(ENV).to receive(:[]).and_call_original
```

Comment:
I'm wondering why we need this line. AFAIK we are just stubbing a particular env var in the line below and not affecting the rest of ENV.

Reply:
I don't see this done in other specs.

Reply:
We stub the call of `[]` on ENV. But in a feature spec, a lot more variables are accessed than the ones we stub here. I got an error that call of something like `ENV['TMP_DIR']` was not expected. The test doesn't want to care about all the other environment variables and therefore I put this catch all in. Is there a better way?